nncf/torch/layers.py

```diff
@@ -385,6 +386,7 @@ def from_module(module):
 NNCF_WRAPPED_USER_MODULES_DICT = {}
 
 
+@api(canonical_alias="nncf.torch.load_state")
 def register_module(*quantizable_field_names: str, ignored_algorithms: list = None):
```

Comment:
Nevertheless is it `load_state` or `register_module`?

Reply:
Fixed, and added a check in `conf.py` to guard against this.